### PR TITLE
[zh]Add 2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fix:#32407
Translate 2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md into Chinese

### CODE REVIEW

File: content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+<!--
+Caleb Woodbine is currently a member of the ii.nz organisation.
+-->
+Caleb Woodbine 目前是 ii.nz 组织的成员。
+
+<!--
+He began contributing to the Kubernetes project in 2018 as a member of the Kubernetes Conformance working group. His experience was positive, and he benefited from early guidance from [Hippie Hacker](https://github.com/hh), a fellow contributor from New Zealand.
+-->
+他于 2018 年作为 Kubernetes 一致性工作组的成员开始为 Kubernetes 项目做贡献。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md:65]:
```suggestion
他于 2018 年作为 Kubernetes Conformance 工作组的成员开始为 Kubernetes 项目做贡献。
```

File: content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+<!--
+He has made major contributions to Kubernetes project since then through `SIG k8s-infra` and `k8s-conformance` working group.
+-->
+他在 `SIG k8s-infra` 和 `k8s-conformance` 工作组为 Kubernetes 项目做出了重大贡献。
+
+<!--
+Caleb is also a co-organizer of the [CloudNative NZ](https://www.meetup.com/cloudnative-nz/) community events, which aim to expand the reach of Kubernetes project throughout New Zealand in order to encourage technical education and improved employment opportunities.
+-->
+Caleb 还是 [CloudNative NZ](https://www.meetup.com/cloudnative-nz/)
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md:76]:
```suggestion
Caleb 也是 [CloudNative NZ](https://www.meetup.com/cloudnative-nz/)
```

File: content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+<!--
+He believes that consistent attendance at community/project meetings, taking on project tasks, and seeking community guidance as needed can help new aspiring developers become effective contributors.
+-->
+他相信，持续参加社区/项目会议，承担项目任务，并在需要时寻求社区指导，可以帮助有抱负的新开发人员成为有效的贡献者。
+
+<!--
+> _The feeling of being a part of a large community is really special. I've met some amazing people, even some before the pandemic in real life :)_
+-->
+> _成为大社区的一份子感觉真的很特别。我遇到了一些了不起的人，甚至有些人出现在现实生活中。_
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md:110]: pandemic 疫情

File: content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+<!--
+He recommends that new contributors use pair programming.
+-->
+他建议新的贡献者结对编程。
+
+<!--
+> _The cross pollination of approaches and two pairs of eyes on the same work can often yield a much more amplified effect than a PR comment / approval alone can afford._
+-->
+> _针对一个项目，多人关注和交叉交流往往比单独的评论/认可 PR 能产生更大的效果。_
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md:140]:
```suggestion
> _针对一个项目，多人关注和交叉交流往往比单独的评审、批准 PR 能产生更大的效果。_
```

File: content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+## [Nick Young](https://github.com/youngnick)
+
+<!--
+Nick Young works at VMware as a technical lead for Contour, a CNCF ingress controller. He was active with the upstream Kubernetes project from the beginning, and eventually became the chair of the LTS working group, where he advocated user concerns. He is currently the SIG Network Gateway API subproject's maintainer.
+-->
+Nick Young 在 VMware 工作，是 CNCF 入口控制器 Contour 的技术负责人。
+他从一开始就积极参与上游 Kubernetes 项目，最终成为 LTS 工作组的主席，
+他提倡关注用户。他目前是 SIG 网络网关 API 子项目的维护者。
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-03-15-meet-our-contributors-APAC-AU-NZ-region-01.md:149]:
```suggestion
他提倡关注用户。他目前是 SIG Network Gateway API 子项目的维护者。
```

**Comment by zhangxyjlu** [comment]
I have revised all the review comments. Thank you@[tengqm](https://github.com/tengqm)

**Comment by tengqm** [comment]
please squash your commits into one.

**Comment by zhangxyjlu** [comment]
> please squash your commits into one.

I have rebased it. Thanks for your review.@[tengqm](https://github.com/tengqm)

**Comment by mengjiao-liu** [comment]
/lgtm
Thanks!

**Pull request merged**